core/includes/database/mysql/schema.inc

```diff
@@ -291,31 +312,30 @@ class DatabaseSchema_mysql extends DatabaseSchema {
     return implode(', ', $return);
   }
 
-  protected function createKeysSqlHelper($fields) {
```

Comment:
`createKeySql()` and `createKeySqlHelper()` were identical methods. Because this only exists within the MySQL schema class and it is protected (and it is unusual), I think it is safe to remove.

Reply:
Removing a method is an interesting move in a "coding standards only" PR. :wink: 

But it is/was obviously the exact same code (it was there since 2008). And removing a protected method in a class that nobody extends ... wait a minute... the Schema module extends DatabaseSchema_mysql.
But the module doesn't fiddle with key creation. Still OK to remove this method. :wink: